app/code/Magento/Fedex/Model/Carrier.php

```diff
@@ -1571,7 +1571,7 @@ private function processTrackingDetails(\stdClass $trackInfo)
         ];
 
         if (!empty($trackInfo->ShipTimestamp)) {
-            $datetime = \DateTime::createFromFormat(\DateTime::ISO8601, $trackInfo->ShipTimestamp);
+            $datetime = \DateTime::createFromFormat('U', strtotime($trackInfo->ShipTimestamp));
```

Comment:
You don't need to do any changes because this issue already fixed by Magento team and will be delivered soon.